### DISCUSSION TITLE
Redirecting old images

### DISCUSSION THREAD

**Discussion by nathanaelphilip** [vercel/next.js#28266] [Help] [answered]
We’ve moved a wordpress site to use NextJS + Vercel, but we’re getting a lot of errors from old images. I tried to set up redirects, but anything with a suffix seems to just error out – image.jpg, wp-login.php;

Is there a way around this?

**Comment by nathanaelphilip** [comment]
example: https://invoice.2go.com/img/logo-opengraph.png

**Reply by leerob** [reply]
Are you using a catch-all route? It seems like it's trying to invoke a function here and run some code, rather than throwing a 404 for the asset not being found.

**Reply by nathanaelphilip** [reply]
yeah – but all non `.png` routes redirect properly – I can try to throw the 404 on my end if that’s the solution – I think I just need to fix the way the api handles those requests. Thanks @leerob :)

**Comment by nathanaelphilip** [comment, marked as answer]
The CMS server was having a fit with the URLs, so I just had to handle the error on the NextJS side. Thanks again @leerob
